crates/driver/src/infra/config/file/mod.rs

```diff
@@ -343,10 +343,6 @@ enum Account {
     PrivateKey(eth::B256),
     /// AWS KMS is used to sign transactions. Expects the key identifier.
     Kms(#[serde_as(as = "serde_with::DisplayFromStr")] Arn),
-    /// An address is used to identify the account for signing, relying on the
-    /// connected node's account management features. This can also be used to
-    /// start the driver in a dry-run mode.
-    Address(eth::Address),
```

Comment:
This is a breaking change. For testing purposes it's really nice to be able to plug in a public address. This ensures that the driver will not be able to submit a transaction while you are experimenting (avoid loss of money).
Additionally this allows you to role play as an already allow listed account to bypass the authentication contract.

Reply:
I see, will add one then. It will error out anyway on the alloy side because node-side signing goes through a different path in alloy

Reply:
Done: https://github.com/cowprotocol/services/pull/3995/changes#diff-952335a51c49ec541b3054df3f0f424e3f304d1becf6ad93de9f9735fe8409b1R134-R137

cc @m-sz